src/subcommand/server.rs

```diff
@@ -232,6 +232,10 @@ impl Server {
           "/r/children/:inscription_id/inscriptions",
           get(Self::child_inscriptions_recursive),
         )
+        .route(
+          "/r/children/:inscription_id/inscriptions/at/:index",
```

Comment:
We dont need `/inscriptions/` in the route it is too verbose and is plural. We are only returning a single Id here.
We should follow the format of the first child endpoint not the second one. I never got around to doing this feature on the original child endpoint I was going to but ran out of time.
```diff
-        "/r/children/:inscription_id/inscriptions/at/:index",
+        "/r/children/:inscription_id//at/:index",
```

Reply:
sounds good.

Reply:
done.